src/Umbraco.Web.UI.Client/src/packages/tags/components/tags-input/tags-input.element.ts

```diff
@@ -78,13 +78,23 @@ export class UmbTagsInputElement extends UUIFormControlMixin(UmbLitElement, '')
 		this._matches = data.items;
 	}
 
-	#onKeydown(e: KeyboardEvent) {
+	#onKeydown(e: KeyboardEvent, idx: number) {
 		//Prevent tab away if there is a input.
 		if (e.key === 'Tab' && (this._tagInput.value as string).trim().length && !this._matches.length) {
```

Comment:
Move `(this._tagInput.value as string).trim().length` into a const, so that one can be used in both if-sentences :-)